src/sentry/static/sentry/app/views/planout/installReact.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import createReactClass from 'create-react-class';
+
+import {t} from 'app/locale';
+
+const InstallReactTest = createReactClass({
+  displayName: 'InstallReactTest',
+
+  propTypes: {
+    dsn: PropTypes.string,
+  },
+
+  getInitialState(props) {
+    return {
+      loading: true,
+      error: false,
+    };
+  },
+
+  render() {
+    let dsn = this.props.dsn;
+
+    return (
+      <div>
+        <h3> Installation </h3>
+        <p>
+          {t('Start by adding the')}
+          <span className="pre"> raven.js </span>{' '}
+          {t(`script tag to your
+          page. It should be loaded as early as possible, before your main javascript
+          bundle.`)}
+        </p>
+        <pre>
+          <span>{'<'}</span>
+          <ScriptSpan>script </ScriptSpan>
+          <AttributeSpan>src</AttributeSpan>
+          <span>=</span>
+          <ValueSpan>{t('https://cdn.ravenjs.com/3.24.0/raven.min.js')}</ValueSpan>
+          <br />
+          <AttributeSpan> crossorigin</AttributeSpan>
+          <span>=</span>
+          <ValueSpan>{'anonymous'}</ValueSpan>
+          <span>{'></'}</span>
+          <ScriptSpan>script</ScriptSpan>
+          <span>{'>'}</span>
+        </pre>
+
+        <h3> Basic Configuration </h3>
+        <p>{t('Next configure Raven.js to use your Sentry DSN: ')}</p>
```

Comment:
I'd love to flesh this out a bit more. We're suggesting the configure environment and release—perhaps a few words on *why* this is useful and link to the relevant docs.

Reply:
Not a blocking comment though—we can ship and keep adding.